### ISSUE TITLE
Can not get it to work

### ISSUE THREAD

**Issue by lenton** [BradCrumb/atom-linter-phpstan#6]
I'm new to Atom so apologies if this is my fault. I've tried to set up this package but doesn't appear to be working at all. There's also no errors in the developer tools pane, so nothing to give an indication of why it isn't working.

I'm using this version of Atom:

```bash
$ atom --version
Atom    : 1.35.1
Electron: 2.0.18
Chrome  : 61.0.3163.100
Node    : 8.9.3
```

I have PHPStan installed both globally and locally in my project:

```bash
$ phpstan --version
PHPStan - PHP Static Analysis Tool 0.11.3
$ php vendor/bin/phpstan --version
PHPStan - PHP Static Analysis Tool 0.11.1
```

I've installed the package into Atom, and it's enabled. The settings of the package have been left on the defaults, as I understand it searches automatically for a project configuration file.

My PHPStan config file is located at my project root:

```bash
$ cat phpstan.neon
parameters:
    bootstrap: %currentWorkingDirectory%/includes/bootstrap.php
    paths:
        - %currentWorkingDirectory%/src
    level: 7
```

If I run PHPStan manually I get `[ERROR] Found 1828 errors`, yet none of these are appearing in Atom.

Any idea on what I've done wrong? Do you have any suggestions of what commands I can run to help diagnose the issue?

**Comment by BradCrumb** [comment]
@lenton Sorry for my very late response. I will fix issue #7 as soon as possible.

**Comment by BradCrumb** [comment]
@lenton  Just released a new version with a fix for this issue. Credits to @mallardduck .
